datafusion/physical-plan/src/execution_plan.rs

```diff
@@ -546,6 +558,23 @@ pub trait ExecutionPlan: Debug + DisplayAs + Send + Sync {
             child_pushdown_result,
         ))
     }
+
+    /// Whether this operator supports cooperative yielding. Default is false.
+    fn yields_cooperatively(&self) -> bool {
+        false
```

Comment:
Shouldn't this use the result of `emission_type` to return the default value instead of just returning `false`?